MediaRecorder.html

```diff
@@ -648,17 +703,24 @@
     <h2>Error Handling</h2>
     <section id="general-principles">
       <h3>General Principles</h3>
-      <p>The UA MUST throw a <a href= "https://heycam.github.io/webidl/#idl-
-      DOMException">DOMException</a> (see [[!DOM4]]) when the error can be
-      detected at the time that the call is made. In all other cases, an
-      <a><code>ErrorEvent</code></a> MUST be fired. If recording has been
-      started and not yet stopped when the error occurs, then after raising the
-      error, the UA MUST fire an event named <code>dataavailable</code>,
-      containing any data that it has gathered, and then a <code>stop</code>
-      event. The UA may set platform-specific limits, such as those for the
-      minimum and maximum Blob size that it will support, or the number of
-      Tracks it will record at once. It must signal a fatal error if these
-      limits are exceeded.</p>
+
+      <p>The UA MUST throw a <a
+      href="https://heycam.github.io/webidl/#idl-DOMException">DOMException</a>
+      ([[!DOM4]]) when the error can be detected at the time that the call is
```

Comment:
Not sure why this says DOM4, the link is right in that WebIDL defines DOMException now.